Update load-balancer-standard-overview.md to reflect LB type to add to regain outbound snat

dariopb opened the pull request:
Load Balancer to be added to enable outbound snat is Basic (instead of standard as was before).

azurecla commented:
Hi __@rjmax__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ryjones). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

IlyaGrebnov commented on articles/azure-subscription-service-limits.md:
```diff
@@ -165,6 +174,38 @@ This document specifies some of the most common Microsoft Azure limits. Note tha
+
+<table cellspacing="0" border="1">
+<tr>
+   <th align="left" valign="middle">Resource</th>
```
Information is outdated. It is 800 actually :)

rjmax replied:
Fixed :)

yangl900 commented on articles/azure-subscription-service-limits.md:
```diff
@@ -97,7 +97,16 @@ This document specifies some of the most common Microsoft Azure limits. Note tha
-
+<tr>
+   <td valign="middle"><p>Resource Manager API Reads</p></td>
+   <td valign="middle"><p>3200 per hour</p></td>
```
typo. this should be 32000 per hour.

dariopb committed:
Update load-balancer-standard-overview.md

Load Balancer to be added to enable outbound snat is Basic (instead of standard as was before).
